pkg/buildscript/raw.go

```diff
@@ -1,19 +1,13 @@
 package buildscript
 
 import (
-	"strconv"
-	"strings"
-	"time"
-
-	"github.com/ActiveState/cli/internal/rtutils/ptr"
 	"github.com/brunoga/deep"
 )
 
 // Tagged fields will be filled in by Participle.
 type rawBuildScript struct {
+	Info        *string       `parser:"(RawString @RawString RawString)?"`
```

Comment:
That sounds like a task for another day when we replace project files with build scripts, because right now, callers of `buildscript.Unmarshal()` expect either a full, populated build script, or an error explaining why it didn't get one. We would have to refactor all callers to handle receiving a build script with only checkout info, or a fully populated one. Even still, I don't think they could do anything meaningful with a script that only has checkout info. (When we replace project files with build scripts, I can see us having the primer or some really early function make use of this to set things up, but then perhaps if that's the only use, it should be in a separate function from `buildscript.Unmarshal()`, which can keep its existing behavior.)

Regardless, I feel like this is a rabbit hole and we shouldn't prescribe something that's not currently needed since we don't know the future requirements yet.

Reply:
Ok, that's fair. We'll leave it for when we get rid of the project URL in activestate.yaml.